[BUG] yarn lint is failing

**Describe the bug**
```yarn lint``` is failing with following error:

```
$ yarn lint
yarn run v1.22.19
$ yarn tsc --noEmit && yarn lint-es && yarn lint-sass
$ /home/ubuntu/OSD/oui/node_modules/.bin/tsc --noEmit
$ eslint --cache "{src,src-docs}/**/*.{ts,tsx,js}" --max-warnings 0

/home/ubuntu/OSD/oui/src/components/icon/icon.test.tsx
  34:21  error  Unable to resolve path to module 'cheerio'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)
```

Because of failed ```yarn-lint```, I'm not able to commit the changes.

**To Reproduce**
Steps to reproduce the behavior:
Run ```yarn lint```

**Expected behavior**
```yarn lint``` should be successful.

**OpenSearch Version**
N/A

**Dashboards Version**
N/A

**Plugins**

N/A



**Host/Environment (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser and version [e.g. 22]

**Additional context**

Add any other context about the problem here.

--- a/package.json
+++ b/package.json
@@ -58,6 +58,8 @@
     "**/eslint/shelljs": "^0.8.5",
     "**/css-select/css-what": "^6.1.0",
     "**/css-select/nth-check": "^2.0.1",
+    "**/cheerio-select-tmp/css-what": "^6.1.0",
+    "**/cheerio-select-tmp/nth-check": "^2.0.1",
     "**/find-versions": "^4.0.0",
     "codesandbox/**/make-fetch-happen": "^4.0.1",
     "codesandbox/**/pacote": "^12.0.0",

